compiler_gym/envs/llvm/service/LlvmEnvironment.cc

```diff
@@ -188,7 +188,13 @@ void LlvmEnvironment::runPass(llvm::Pass* pass, ActionReply* reply) {
   llvm::legacy::PassManager passManager;
   setupPassManager(&passManager, pass);
 
-  const bool changed = passManager.run(benchmark().module());
+  bool changed = passManager.run(benchmark().module());
+
+  // TODO: Fix changed. changed is not reliable in LLVM as currently
+  // implemented and the gym environment relies on it being accurate,
+  // so conservatively hardcode it to true for now.
+  changed = true;
```

Comment:
Done.

Just for posterity, this is the command line I used to run the microbenchmarks:

  bazel test --test_output=all --test_arg=-s :bench_test --nocache_test_results --test_arg=-k='test_step and cBench-v0 and adpcm'

Results here:

  https://pastebin.com/raw/81esBiVw

The difference, if any, is in the noise, which makes sense since I placed some CHECK(false) and determined that eager reward and observation are not turned on for that microbenchmark. One could add a microbenchmark for this, but not clear if it's actionable for this change now.